lms/djangoapps/courseware/access.py

```diff
@@ -160,11 +181,10 @@ def _can_access_descriptor_with_start_date(user, descriptor, course_key):  # pyl
             descriptor,
             course_key=course_key
         )
-        return (
-            descriptor.start is None
-            or now > effective_start
-            or in_preview_mode()
-        )
+        if descriptor.start is None or now > effective_start or in_preview_mode():
```

Comment:
For readability, either keep this as a multi-line statement or add intermediate parenthesis.